dmt/main/models.py

```diff
@@ -710,6 +710,31 @@ def all_actual_times(self):
             item__milestone__project=self,
         ).order_by('completed')
 
+    def users_active_between(self, start, end):
```

Comment:
I rewrote this function using django's db aggregation features. The function is actually simpler now.. thanks for pointing me in the right direction!

I removed all the test skips and they are all passing with a local `make jenkins`, so this PR is ready for further review/merge.